pyproject.toml

```diff
@@ -5,12 +5,13 @@ build-backend = "poetry.masonry.api"
 [tool.poetry]
 name = "yapapi"
 version = "0.2.0"
-description = "Hi-Level API for Golem The Next Milestone"
+description = "High-Level API for Golem The Next Milestone"
```

Comment:
Not sure if "Golem The Next Milestone" is okay.

Reply:
hmm, maybe indeed, it should be "the New Golem" ... ? ...